### ISSUE TITLE
Is it possible to nest ternary operator?

### ISSUE THREAD

**Issue by mariusst** [saltcorn/saltcorn#1565]
Hi,

Is it possible to nest ternary operator inside calculated fields? Something like:
`condition1 ? value1
        : condition2 ? value2
        : condition3 ? value3
        : value4`

**Comment by glutamate** [comment]
yes, this is valid JavaScript and therefore a valid formula. I do this all the time!
